torch/_inductor/ir.py

```diff
@@ -2219,11 +2219,13 @@ def get_fill_order(self):
                 for r in reads
             ]
             # only consider reads to buffer of same size
+            # ignore StarDeps because they don't contribute stride information
             reads = [
                 sympy_subs(
                     r.index, {v: sympy.Integer(0) for v in reduction_vars if v != 0}
                 )
                 for r in reads
+                if isinstance(r, dependencies.MemoryDep)
```

Comment:
nit: maybe assert that that all the reads are `MemoryDep` or `StarDep` ?

Reply:
yeah definitely a good idea